src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -24,6 +26,16 @@ export class CalciteShellPanel {
    */
   @Prop({ reflect: true }) collapsed = false;
 
+  /**
+   * This property makes the content area appear like a "floating" panel.
+   */
+  @Prop({ reflect: true }) contentDetached = false;
```

Comment:
Can we rename to just `detached` and `detachedScale` instead of `contentDetached` and `contentDetachedScale`?

Reply:
I like it.

Reply:
Updated.